ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/export-actions/export-actions.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import { hot } from 'react-hot-loader'
+import styled from '../../styles/styled-components'
+
+const Root = styled<{}, 'div'>('div')`
+  overflow: auto;
+
+  div {
+    cursor: pointer;
+    display: block;
+    line-height: ${props => props.theme.minimumButtonSize};
+    padding: 0px 15px;
+    background-color: #253540;
```

Comment:
❓ How can I pull in the theme's standard colors to use for `background-color`?

Reply:
Probably something along the lines of `background-color: ${props => props.theme.background...}`
https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/styles/styled-components/theme.tsx#L75